src/io/envrionment.go

```diff
@@ -0,0 +1,41 @@
+package io
+
+import (
+	"fmt"
+	"os/exec"
+)
+
+//CheckRequiredPrograms check wether or not the programs are installed on the current machine
+//Used by the script too ensure that the local environment is correctly setup
```

Comment:
already changed "too"  to "to"

Reply:
Setup should be in two words in the case (end of the same line)